platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -23,7 +23,12 @@ const ORIENTATION_MAP = {
 };
 
 function ViewerViewportGrid(props) {
-  const { servicesManager, viewportComponents, dataSource } = props;
+  const {
+    servicesManager,
+    viewportComponents,
+    dataSource,
+    modeViewportPaneStyles = {},
```

Comment:
What is this, there are too many things happening. There is `viewoprtGridProps`, `modeViewportPaneStyles` and there is `customStyle` what is their relationship with each other